### DISCUSSION TITLE
Azure TRE Accelerator Community Meetup 11 Sep 24

### DISCUSSION THREAD

**Discussion by tim-p-allen** [microsoft/AzureTRE#4077] [Events]
## Details

Welcome to the inaugural Azure TRE Accelerator Community Meetup, a monthly virtual gathering that brings together researchers, academics, healthcare providers, data scientists, research software engineers and DevOps from across the Azure Trusted Research Environment (TRE) ecosystem. Our aim is to foster collaboration, share knowledge, and drive innovation in the world of secure, collaborative research environments. 

## Event Details
Date and Time: 11 September followed by every 2nd Wednesday of the month, 16:00 - 17:00 BST  

## Who Should Attend?
This event is perfect for current Azure TRE users, data scientists, researchers, IT professionals managing TREs, and anyone interested in the future of secure data environments. Whether you're an experienced RSE or just getting started, your insights and participation are valuable to our community.

## Why Attend?
By joining our monthly meet, you'll stay informed about and input directly into the latest Azure TRE developments and strategic roadmap. You'll have the opportunity to learn from peers, share your own experiences, and contribute to the direction of Azure TRE development. It's also an excellent opportunity to network with a community of like-minded professionals who are passionate about advancing trusted research environments.

## Agenda
Our monthly meet follows a structured agenda to ensure we cover all crucial aspects of Azure TRE:

1. **Welcome and Introduction**
    - Context for the community meet
    - Overview of the session's agenda
3. **Envisioning a feature rich Azure TRE**
    - The Azure TRE Accelerator
    - Building a community-led framework for a feature rich Azure TRE
4. **Community Spotlight Session**
    - Showcase of innovative customisations and use cases
    - Guest speakers sharing their experiences and best practices
5. **Azure TRE Development highlights**
    - New feature releases
6. **Wrap Up**
    - Summary of key takeaways
    - Information on how to get involved and contribute

## Key Benefits
You'll gain a collective understanding of the strategic TRE roadmap directly from experts and decision-makers. Our community sharing sessions will showcase new developments and customisations from other users and contributors, helping you discover best practices and use cases to enhance your own Azure TRE implementation. Moreover, you'll have the chance to influence development by contributing your ideas and feedback to shape the future of Azure TRE.

## How to Participate
We encourage active participation throughout the session. Feel free to ask questions and share your experiences during the meet. The community spotlight session offers an excellent opportunity to showcase your own Azure TRE innovations. You can also engage with speakers and other attendees through the Teams chat function, fostering a dynamic and interactive environment.

## Registration
To join us in shaping the future of secure, collaborative research environments with Azure TRE, please register on [this page](https://events.teams.microsoft.com/event/a7518c7c-3645-46cc-9613-6bb321159e53@1fb496e0-1eea-4fbd-be38-d4b9cda1d791).

We look forward to welcoming you to our vibrant community, together, we'll explore the cutting edge of trusted research environments and drive the future of secure, collaborative research.

**Comment by CCCowley** [comment]
Hi everyone,

Thank you to everyone that came along to Wednesday's meet-up, it was a fantastic gathering of minds from across the research and technology spectrum. For those who couldn't make it, here's a quick rundown of my personal highlights.

This inaugural community meet-up session really highlighted how the community is pushing the envelope of what's currently possible with AzureTRE and TREs in general.

It was really interesting to hear about the expansion and adoption of TREs beyond higher education and healthcare with Nick Woodley shared some great examples of TRE applications in finance and insurance. It's exciting to see the technology finding new homes in different industries.

The OpenAI integration demos were a definite highlight. @harryy94 and Ben Roberts showed us some impressive work that's gone into opening up new possibilities for researchers with LLMs. 

@tim-allen-ck  gave us a helpful update on the latest TRE release. He made a good point about the importance of community contributions - there are plenty of ways for everyone to get involved, from raising GitHub issues to contributing code.

We also heard from Michal Rosiek about King's College London's experience building a TRE from scratch. The discussion around balancing security and usability is something I'm sure will be familiar to many and possibly worth exploring in future meet-ups.

I've put together a quick survey to get people's ideas on how to keep improving these meet-ups and to start setting an agenda for next months meet-up: [https://forms.office.com/e/r12GubjKgY](https://forms.office.com/e/r12GubjKgY ). We'll share the full results with the community too.

If you would be interested in contributing to a future meet-up, please do reach out!

The full recording can be found below, hope to see many of you at the next one!

https://youtu.be/N8vj4pMMkI0?si=S8HK5ZIx_LbgPkrq
